### DISCUSSION TITLE
Install kepler per node or per cluster ?

### DISCUSSION THREAD

**Discussion by jochen-schuettler** [sustainable-computing-io/kepler#1373] [Q&A]
My understanding was kepler needs to be installed only once per cluster, but I was told it's once per node instead. Who is right? Also where in installation docs is the info on that?

**Comment by zhifanl** [comment]
cluster

**Comment by iajellabi** [comment]
Kepler is deployed as a Daemonset ( once deployed, it will be on every node of the cluster ), so it is per cluster !
